torch/csrc/jit/passes/quantization.cpp

```diff
@@ -145,6 +145,8 @@ class InsertObserversHelper {
   // Values that are the output of GetAttr[name="bias"] and they
   // will be propagated through the function call hierarchy
   std::unordered_set<Value*> bias_values_;
+  // Unique id generator for observer module
```

Comment:
Nit: the comment might cover why we do this and when this is used, why we put it here and not right before the loop. Basically, cover more of "why" than of "what".